.github/workflows/pull-request-ci.yaml

```diff
@@ -199,14 +199,21 @@ jobs:
           echo "::set-env name=SSH_AUTH_SOCK::${SSH_AUTH_SOCK}"
           echo "::set-env name=SSH_AGENT_PID::${SSH_AGENT_PID}"
 
-      # set GKE secret. TBD: use vault instead
+      - name: Query vault for GKE secrets
+        id: vault
+        uses: hashicorp/vault-action@v2.0.1
+        with:
+          url: https://volt.cap.explore.suse.dev/
+          token: ${{ secrets.VAULT_TOKEN }}
```

Comment:
Hmm, there's a `github` auth method; if that works, we won't need to set a token (that we'd need to rotate), since `github.token` is automatic.  Not sure _who_ that token identifies, though; so if it doesn't use the right one out of the box there's no point (i.e. setting a token for a user and manually setting it as a secret has no point).

Reply:
This is the same token @viovanov set up for pxc-docker. And no, this is not github token. I remember, I thought that it was GH token and that was then the first issue the GHA flow for pxc-docker had.

Reply:
if it's a github token, it would suffice for it to be a read:org GH token for an account that is on the corresponding team, (maybe `suse-platform-team` GH team?).

Reply:
Tried it over in #1592; it turns out trying to use the Actions token results in a HTTP 500 from Vault instead (probably because it had problems resolving that token to an actual person).  So that's not going to work.  Marking this as resolved.